### ISSUE TITLE
New file structure + /lib/download.sh

### ISSUE THREAD

**Issue by ffd8** [basiljs/basil.js#229]
Unfortunately I missed the discussion #155 about the `basiljs/bundle` – with a newly proposed structure:

`basiljs/basil.js`
`basiljs/examples`
`basiljs/libraries`
...

Where will `download.sh` go? `/libraries` folder?  
`/bundle` was great for masking such things.

Unfortunately removing `/bundle` breaks all previous scripts until person's header path is fixed... making sure it's worth it? (becomes main issue if b.less isn't a thing and most scripts would work).

**Comment by trych** [comment]
@ffd8 Yes, but that was actually the structure we agreed on in our Skype call last year. 😃

Yes, the header needs to change once, but that's the backwards incompatible change that we agreed on. And from my experience in using this new bundled version in class, I can tell you it is _so_ much nicer to work with this bundled version that I think it was a really important move to have a user friendly workflow with basil.

Btw. the download.sh script gets generated on the fly as soon as you call the `b.download()` function. It gets put in a `lib` folder next to your `basil.js` file. Having said that, I wonder however, why we need a `download.sh` file at all and don't just generate the script string on the fly, execute it via `b.shell()` and never write it to disk in the first place. That would work just as well, "leave no traces" and leave the basil.js file even more flexible.
